### PR TITLE
OCPBUGS-18522: removes ovn-kubernetes-master from 4.13

### PR DESCRIPTION
**Pull request by JoeAldinger**
Version(s):
4.13

Issue:
https://issues.redhat.com/browse/OCPBUGS-18522

Link to docs preview:
Preview [link](https://65373--docspreview.netlify.app/openshift-enterprise/latest/networking/ovn_kubernetes_network_provider/ovn-kubernetes-architecture-assembly#:~:text=3%20more...%20%20%2019m-,NAME,-DATA%20%20%20AGE%0Aconfigmap)

QE review:
- [x] QE has approved this change.

Additional information:

### CODE REVIEW

File: modules/nw-ovn-kubernetes-list-resources.adoc [hunk 1]
```diff
@@ -96,4 +94,18 @@ $ oc get pods ovnkube-node-jg52r \
 ovn-controller ovn-acl-logging kube-rbac-proxy kube-rbac-proxy-ovn-metrics ovnkube-node
 ----
 +
-The `ovnkube-node` pod has a container (`ovn-controller`) that resides on each {product-title} node. Each node’s `ovn-controller` connects the OVN northbound to the OVN southbound database to learn about the OVN configuration. The `ovn-controller` connects southbound to `ovs-vswitchd` as an OpenFlow controller, for control over network traffic, and to the local `ovsdb-server` to allow it to monitor and control Open vSwitch configuration.
\ No newline at end of file
+The `ovnkube-node` pod has a container (`ovn-controller`) that resides on each {product-title} node. Each node’s `ovn-controller` connects the OVN northbound to the OVN southbound database to learn about the OVN configuration. The `ovn-controller` connects southbound to `ovs-vswitchd` as an OpenFlow controller, for control over network traffic, and to the local `ovsdb-server` to allow it to monitor and control Open vSwitch configuration.
+
+. List the OVN-Kubernetes elected leader by running the following command:
```
Comment (anuragthehatter) [inline modules/nw-ovn-kubernetes-list-resources.adoc:99]: @tssurya Does this stmt seems ok to you? Or we need something like kubernetes elected OVN leader?

rest lgtm

File: modules/nw-ovn-kubernetes-list-resources.adoc [hunk 2]
```diff
@@ -25,45 +25,43 @@ $ oc get all,ep,cm -n openshift-ovn-kubernetes
...
-pod/ovnkube-master-lqs4v   6/6     Running   0             57m
-pod/ovnkube-master-vxhtq   6/6     Running   0             57m
-pod/ovnkube-node-9k9kc     5/5     Running   0             57m
-pod/ovnkube-node-jg52r     5/5     Running   0             51m
-pod/ovnkube-node-k8wf7     5/5     Running   0             57m
-pod/ovnkube-node-tlwk6     5/5     Running   0             47m
-pod/ovnkube-node-xsvnk     5/5     Running   0             57m
+NAME                       READY   STATUS    RESTARTS        AGE
+pod/ovnkube-master-2qnbv   6/6     Running   0               19m
+pod/ovnkube-master-kqmnv   6/6     Running   1 (11m ago)     19m
```
Comment (tssurya) [inline modules/nw-ovn-kubernetes-list-resources.adoc:30]: why are these bits changing? we are showing an output with more restarts :P just make them 0 restarts or do we really care about pod names on docs?

File: modules/nw-ovn-kubernetes-list-resources.adoc [hunk 3]
```diff
@@ -25,45 +25,43 @@ $ oc get all,ep,cm -n openshift-ovn-kubernetes
...
+pod/ovnkube-node-jcdh6     5/5     Running   1 (8m50s ago)   9m33s
+pod/ovnkube-node-xrml8     5/5     Running   1 (18m ago)     19m
 
 NAME                            TYPE        CLUSTER-IP   EXTERNAL-IP   PORT(S)             AGE
-service/ovn-kubernetes-master   ClusterIP   None         <none>        9102/TCP            57m
-service/ovn-kubernetes-node     ClusterIP   None         <none>        9103/TCP,9105/TCP   57m
-service/ovnkube-db              ClusterIP   None         <none>        9641/TCP,9642/TCP   57m
+service/ovn-kubernetes-master   ClusterIP   None         <none>        9102/TCP            19m
+service/ovn-kubernetes-node     ClusterIP   None         <none>        9103/TCP,9105/TCP   19m
+service/ovnkube-db              ClusterIP   None         <none>        9641/TCP,9642/TCP   19m
```
Comment (tssurya) [inline modules/nw-ovn-kubernetes-list-resources.adoc:42]: only AGE is different here?

File: modules/nw-ovn-kubernetes-list-resources.adoc [hunk 4]
```diff
@@ -25,45 +25,43 @@ $ oc get all,ep,cm -n openshift-ovn-kubernetes
...
-daemonset.apps/ovnkube-node     5         5         5       5            5           beta.kubernetes.io/os=linux                                   57m
+daemonset.apps/ovnkube-master   3         3         3       3            3           beta.kubernetes.io/os=linux,node-role.kubernetes.io/master=   19m
+daemonset.apps/ovnkube-node     6         6         6       6            6           beta.kubernetes.io/os=linux                                   19m
 
-NAME                              ENDPOINTS                                                        AGE
-endpoints/ovn-kubernetes-master   10.0.132.11:9102,10.0.151.18:9102,10.0.192.45:9102               57m
-endpoints/ovn-kubernetes-node     10.0.132.11:9105,10.0.143.72:9105,10.0.151.18:9105 + 7 more...   57m
-endpoints/ovnkube-db              10.0.132.11:9642,10.0.151.18:9642,10.0.192.45:9642 + 3 more...   57m
+NAME                              ENDPOINTS                                               AGE
+endpoints/ovn-kubernetes-master   10.0.0.3:9102,10.0.0.4:9102,10.0.0.5:9102               19m
```
Comment (tssurya) [inline modules/nw-ovn-kubernetes-list-resources.adoc:49]: I fail to understand why the outputs here are being changed.

File: modules/nw-ovn-kubernetes-list-resources.adoc [hunk 5]
```diff
@@ -25,45 +25,43 @@ $ oc get all,ep,cm -n openshift-ovn-kubernetes
...
+endpoints/ovn-kubernetes-node     10.0.0.3:9105,10.0.0.4:9105,10.0.0.5:9105 + 9 more...   19m
+endpoints/ovnkube-db              10.0.0.3:9642,10.0.0.4:9642,10.0.0.5:9642 + 3 more...   19m
 
 NAME                                 DATA   AGE
-configmap/control-plane-status       1      55m
-configmap/kube-root-ca.crt           1      57m
-configmap/openshift-service-ca.crt   1      57m
-configmap/ovn-ca                     1      57m
-configmap/ovn-kubernetes-master      0      55m
```
Comment (tssurya) [inline modules/nw-ovn-kubernetes-list-resources.adoc:57]: so far in this change, only this line has really changed :)

File: modules/nw-ovn-kubernetes-list-resources.adoc (hunk 1 above)
Comment (tssurya) [reply modules/nw-ovn-kubernetes-list-resources.adoc:99]:
```suggestion
. List the current OVN-Kubernetes master leader pod by running the following command:
```

**Review by tssurya — CHANGES_REQUESTED** [review]

File: modules/nw-ovn-kubernetes-list-resources.adoc (hunk 4 above)
Comment (JoeAldinger) [reply modules/nw-ovn-kubernetes-list-resources.adoc:49]: Changing back

**Review by tssurya — APPROVED** [review]
/lgtm

**Review by opayne1 — COMMENTED** [review]
/lgtm

**Pull request merged**